homepairs/homepairsUI/src/Screens/Main/Properties/TenantPropertyScreen/TenantPropertyScreenBase.tsx

```diff
@@ -114,7 +114,8 @@ export default function TenantPropertyScreenBase(props:Props){
                     style={Platform.OS === 'web' ? styles.homepairsPropertiesImageWeb : styles.homepairsPropertiesImage}
                     resizeMode='cover'/>
                     </View>
-                    <GeneralHomeInfo property={property} primaryColorTheme={primaryColorTheme} hasEdit={hasEdit} />
+                    <GeneralHomeInfo property={property} primaryColorTheme={primaryColorTheme} hasEdit={canEditProps} />
```

Comment:
Good example here. primaryColorTheme was apart of the DarkMode. Since we are not using darkMode, you can go ahead and remove this.